legal-api/src/legal_api/models/business.py

```diff
@@ -222,7 +222,7 @@ def validate_identifier(identifier: str) -> bool:
         # TODO This is not correct for entity types that are not Coops
         if identifier[:-7] not in ('CP', 'XCP'):
             return False
-
+        """
```

Comment:
I think instead of removing the functionality of validate_identifier, we could just not call it if the filing type is incorporation. That way we still get the validation when we need it.

Reply:
Added NR for now